src/components/atoms/ProjectSearchHeadContent/ProjectSearchHeadContent.tsx

```diff
@@ -75,7 +77,12 @@ export const ProjectSearchHeadContent = (props: ProjectSearchHeadContentProps) =
       },
     ]
     if (isAuthed) {
-      menuList.push({ dataTestId: `export-project`, icon: <BookmarkPlusIcon />, text: t(`projectMenu.addToWatchlists`), onClick: () => hanldeAddToWatchlist(projectWarehouseId) })
+      menuList.push({
+        dataTestId: `export-project`,
```

Comment:
I think we need to use a different id here because its used already for PDF export. May be "add-to-watchlist" ?